palantir-java-format/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -1,56 +0,0 @@
-[
-{
-  "name":"[Lcom.fasterxml.jackson.databind.deser.Deserializers;"
```

Comment:
old file that was generated with graal 21 - now we are using reachability-metadata: https://github.com/palantir/palantir-java-format/pull/1211/files#diff-08891f153c2557b2f9c4131cb3f735dda8476f2f55ed91e9227b769ebe069991R87